bin/wheresmyconfig.sh

```diff
@@ -1,2 +1,2 @@
-
-echo $(dirname $(readlink $(which off-the-record)))/config/
+app_path=$(dirname $(dirname `readlink -f -- $0`))
```

Comment:
Okay, so this uses `readlink` (-f fails silently) but I'm not sure what the `--` does.. $0 is the name of the script that bash is executing... Then you get the dirname of the dirname?

Could you explain this, please?